README.md

```diff
@@ -26,6 +26,13 @@ Mapbox has built one of the best and most amazing OSS ecosystems. A key componen
 ## Donations
 If you or your organisation has seen value from Maputnik, please consider donating at <https://maputnik.github.io/donate>
 
+## Using
+
+The quickest way is to use docker, even without cloning the repo.  Run this command and open your browser to http://localhost:8888 .  Ctrl+C to stop the server.
```

Comment:
In my opinion the quickest way is to use the online editor and using the CLI is also fast. So I think we don't need to say which is the quickest way. Not every user is familiar with docker.

Reply:
> Thanks @nyurik. A quick question, am I correct in thinking that the 2 stage build means that `node` won't be available in the final image? If so I think we'd want to make a `Dockerfile` and `Dockerfile.prod` (or something like that) because otherwise we won't be able to do development with this image. Currently I use https://github.com/orangemug/maputnik-services for all my development, however any development will probably require an `npm install`.

@orangemug could you describe your dev process?  We could create `Dockerfile.dev` (I would prefer to make .dev a special case, because most of the time Dockerfile is built for production), but we have to be clear about the dev pattern here -- e.g. share project root dir with docker, and let docker create node_modules (might be a bit painful with the correct file ownership) and have a long docker command + options to start it right.  Or we could make docker.dev that is fully node-based, and uses `npm start` as its CMD.

Reply:
Clarification: essentially my confusion is how you build:
* if you build locally, you don't need docker
* if you build in docker, you either copy the entire dir into the docker, in which case it doesn't really matter if it has node or not
* or, you share source files with the docker, in which case node_modules are still local, and docker may mess up the file permissions unless you take extra care to run it as local user.

Reply:
Hey @nyurik all my nodejs development happens in containers, really for the ease of setup between environments, and so I never have to worry about installing multiple versions of nodejs etc... For simplier non-maputnik editor projects I use https://github.com/orangemug/nde, including modules I write, for example https://github.com/mgljs-contrib/icepick-style.

For the maputnik editor I use <https://github.com/orangemug/maputnik-services>, basically it'll give me a full maputnik setup website and all, from a single command `docker-compose up` it mounts the code volume, so I edit all my code locally on my machine. If I need to `npm install` that happens via a `docker-compose exec editor npm install uuid`. It seems 
a pretty robust approach, I've been using this pretty much since I started development with Maputnik. I also use a simpler approach in my day job(s) the the last couple of years.

For the sake of this pull request as long as there is a `Dockerfile.dev` which has `node`/`npm` as well as the other common deps (`git`, `python`) available I'm happy. I can alter <https://github.com/orangemug/maputnik-services> to use that file instead. Happy for you just to just copy over the current `Dockerfile` in `master` (`cp Dockerfile Dockerfile.dev`)

Thanks for the PR and let me know if you have any other questions

Reply:
@orangemug the [node image](https://hub.docker.com/_/node) has python and git built-in. The current Dockerfile uses the `node-slim` and adds those manually, but with the new dockerfile, there is no longer a need to use `node-slim` as the base. Can you just use the standard `node` image in your docker-compose?   If you need to run a command, you can just [run it directly](https://stackoverflow.com/questions/30063907/using-docker-compose-how-to-execute-multiple-commands) from the docker-compose file.  And you clearly don't need the dockerfile's `copy` command because your code stays local -- so why have a separate dockerfile?